chapters/arrays.tex

```diff
@@ -1417,12 +1417,12 @@ \section{Empty Arrays}\label{empty-arrays}
 
 Empty matrices can be constructed with the fill function.  For example:
```

Comment:
We should take the opportunity to also fix the formatting here (also _using_ sounds better to me than _with_):
```suggestion
Empty matrices can be constructed using the \lstinline!fill! function.  For example:
```

Reply:
Agreed.